readme.md

```diff
@@ -20,12 +20,17 @@ $ brew cask install mojibar
 
 Go into your apps folder and select Mojibar (Mac shortcut: use command-space bar, then type Mojibar in the spotlight search field). Mojibar will display in the menubar at the top right corner of your screen.
 
+
+#### Linux
+
+You should install [emoji font](https://github.com/eosrei/emojione-color-font), otherwise not all emojies will be rendered properly.
```

Comment:
yeah man I don't know how to write this :(

let me know if it is emojis or emojies